src/Engine/Graphics/RenderBase.cpp

```diff
@@ -484,3 +653,48 @@ HWLTexture *RenderBase::LoadHwlSprite(const std::string &name) {
     return pD3DSprites.LoadTexture(name);
 }
 
+void RenderBase::SavePCXScreenshot() {
+    size_t zeros_number = 5;
+    std::string screenshot_number = std::to_string(engine->config->settings.ScreenshotNumber.Increment());
+    std::string file_name = "screenshot_" + std::string(zeros_number - std::min(zeros_number, screenshot_number.length()), '0') + screenshot_number + ".pcx";
```

Comment:
this is repeated twice, maybe worth having as a separate method?

Reply:
Im not sure where you mean?

Reply:
right.. sorry i'm blind, nevermind
you actually removed the duplication